### PR TITLE
Upload about.md.partial as about.md

### PR DESCRIPTION
**Pull request by bethanyaconnor**
This PR introduces a number of changes:
1. Upload about.md.partial as about.md so that Crowdin treats the file as Markdown. (the goal of this task)
2. Changes the directory structure of the files uploaded to not include `public/`. The rest of the path is maintained as that seemed like useful context.
3. `csf-transition-guide.md` will now be `csf-transition-guide.md.partial` for translated files

Options I didn't take:
1. I could've kept the config specifying the two files separately. However, to do that the directory structure would either be flat (so translators would just see `about.md` and `csf-transition-guide.md` without the rest of the path) or would have extraneous path information (eg `source/public/...`). 

## Links

- [jira](https://codedotorg.atlassian.net/browse/FND-684)

## Testing story

```
ubuntu@ip-10-0-0-11:~/code-dot-org$ crowdin --config bin/i18n/codeorg_markdown_crowdin.yml --identity bin/i18n/CROWDIN_CODEORG_MARKDOWN_CREDENTIALS.yml upload sources --dryrun
Version 2.0.31 of the CLI client is available. You can download the new version here: https://downloads.crowdin.com/cli/v2/crowdin-cli.zip
educate/curriculum/csf-transition-guide.md
international/about.md
```

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by Hamms — COMMENTED** [review]
what does the sync-out look like now? Does `about.md` on crowdin ever get turned back into a `.partial`?

File: bin/i18n/codeorg_markdown_crowdin.yml [hunk 1]
```diff
@@ -1,18 +1,11 @@
...
-  "source": "/public/international/about.md.partial",
-  # cannot use %file_name% and %file_extension% helpers here, because crowdin
-  # doesn't handle multiple extensions like we'd want
-  "translation": "/i18n/%original_path%/about.%locale%.md.partial",
-}, {
-  "source": "/public/educate/curriculum/csf-transition-guide.md",
-  "translation": "/i18n/%original_path%/%file_name%.%locale%.%file_extension%",
-}]
+  "source": "/i18n/locales/source/markdown/**/*.md",
+  "translation": "/pegasus/sites.v3/code.org/i18n/public/%original_path%/%file_name%.%locale%.md.partial",
```
Comment (bethanyaconnor) [inline bin/i18n/codeorg_markdown_crowdin.yml:10]: @Hamms the `.partial` is added here

Comment (Hamms) [reply bin/i18n/codeorg_markdown_crowdin.yml:10]: oooh, I didn't get that this was applied to both of them

**Review by Hamms — APPROVED** [review]

File: bin/i18n/codeorg_markdown_crowdin.yml (hunk 1 above)
Comment (bethanyaconnor) [reply bin/i18n/codeorg_markdown_crowdin.yml:10]: Yeah, the downside is that `csf-transition-guide.md` will now be `csf-transition-guide.md.partial`. It should be a noop but obviously still unnecessary.

Comment (Hamms) [reply bin/i18n/codeorg_markdown_crowdin.yml:10]: I'm increasingly thinking that we might want to rethink the `.partial` extension and just start thinking of this partial syntax as something we generally support in markdown.

Comment (bethanyaconnor) [reply bin/i18n/codeorg_markdown_crowdin.yml:10]: Oh I like that

**Pull request merged**